dom-events.tar/dom-events/buttons.js

```diff
@@ -2,23 +2,26 @@ function handleClick(event){
   console.log('button clicked');
   console.log(event);
   console.log(event.target);
-  document.querySelector('.click-button')
-  if (!$clickButton) throw new Error('The $clickBtn query failed');
-  element.addEventListener('click',handleClick());
 }
+const element = document.querySelector('.click-button');
+  if (!$clickButton) throw new Error('The $clickBtn query failed');
+  element.addEventListener('click', handleClick);
+
+
 function handleMouseOver(event){
   console.log('button hovered');
   console.log(event);
   console.log(event.target);
-  document.querySelector('.hover-button');
-  if (!$hoverButton) throw new Error('The $hoverButton query failed');
-  element.addEventListener('mouseover',event())
 }
+ const element =document.querySelector('.hover-button');
+  if (!$hoverButton) throw new Error('The $hoverButton query failed');
+  element.addEventListener('mouseover',event)
+
 function handleDoubleClick(event){
   console.log('button doubled-clicked');
   console.log(event);
   console.log(event.target);
-  document.querySelector(.'double-click-button');
-  if (!$dblClickButton) throw new Error('The $dblClickButton query failed');
-  element.addEventListener('dblclick', handleDoubleClick());
 }
+ const element=document.querySelector(.'double-click-button');
```

Comment:
you'll need to use a different variable name on lines 16 and 25.

You can't have multiple const variables with the same name or it will give you an error.